entrypoint.sh

```diff
@@ -12,9 +12,17 @@ cf api "$INPUT_CF_API"
 cf auth "$INPUT_CF_USERNAME" "$INPUT_CF_PASSWORD"
 cf target -o "$INPUT_CF_ORG" -s "$INPUT_CF_SPACE"
 
-if [[ -z "$INPUT_CF_COMMAND" ]]; then
-    cf v3-zdt-push -f "$MANIFEST"
+# If no cf CLI command is set, push app with manifest or vars file.
+if [[ -z "$INPUT_CF_COMMAND" ]]; then  
+  if [[ -r "$INPUT_CF_VARS_FILE" ]]; then 
```

Comment:
Where is `$INPUT_CF_VARS_FILE` coming from?  I'm guessing the environment but am not sure, and I see `$MANIFEST` is explicitly defined above.  Then again, that in turn references `$INPUT_CF_MANIFEST`, and I'm not sure where that or any of these other ones actually coming from...  I can't be _that_ rusty on my shell scripting, but it would appear so! 😆

Reply:
Ah, sorry for the confusion. Any variable prepended with `INPUT_` is injected into the script when the action is run. So, when someone sets up their workflow to use this action, they specify [settings like those in the example](https://github.com/cloud-gov/cg-cli-tools/tree/add-vars-support#sample-workflow). So, `cf_manifest` in the workflow file becomes `$INPUT_CF_MANIFEST` in the script when the workflow is executed.

Reply:
I realized my folly shortly after I submitted the review, I should have looked up a bit more about GitHub actions, but thank you for clarifying this! 😀